openrest4j-api/src/main/java/com/openrest/v1_1/Order.java

```diff
@@ -207,7 +211,8 @@ public Order clone() {
                 (loyalty != null) ? loyalty.clone() : null,
                 (posInfo != null) ? posInfo.clone() : null,
                 chargeRoundingStrategy,
-                restaurantLocationId);
+                restaurantLocationId,
+                serviceFees);

```

Comment:
I think that you need to clone the serviceFees list instead of just sending the same pointer to the list.